supervisor/process.py

```diff
@@ -209,6 +211,9 @@ def spawn(self):
         self._assertInState(ProcessStates.EXITED, ProcessStates.FATAL,
                             ProcessStates.BACKOFF, ProcessStates.STOPPED)
 
+        if delaystart:
+            time.sleep(self.config.startdelay)
```

Comment:
Thank you, I will look into that. My first attempt was to only block in supervisorctl but there, as I remember, I did not have access to the individual processes for programs having num_procs > 1.